src/ProjectOrigin.WalletSystem.Server/ViewModels/AggregatedClaimViewModel.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace ProjectOrigin.WalletSystem.Server.ViewModels;
+
+public record AggregatedClaimViewModel
```

Comment:
Didn't we have this functionality?

Reply:
no, aggregated date was earlier not returned from the repository.